performance/utils/loadFile.ts

```diff
@@ -1,8 +1,8 @@
-declare var window: Window;
+declare var window: any;
```

Comment:
If we can no longer use `Window` here than it means the `performance/tsconfig.json` is no longer used. We should either remove that one or make sure the benchmark runner uses it and than we can keep this typing.

Reply:
I think we can revert both changes is this file. i.e. `declare var window: Window` and `new Request(...`. 
That at least seems to be ok according to typescript. This should than be tested as part of the story to make the benchmarks run in the browser.